spec/features/authentication_pages_spec.rb

```diff
@@ -52,40 +56,50 @@
         it { should have_content('Welcome to Twitter Clone') }
         it { should have_link('Sign in') }
         it { should have_link('Sign up') }
+
+        describe 'screenshot', js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.